MDANSE/Src/MDANSE/Framework/Jobs/RootMeanSquareDeviation.py

```diff
@@ -150,20 +150,27 @@ def finalize(self):
         """
         Finalize the job.
         """
+        n_atms = self.configuration["atom_selection"].get_total_natoms()
 
-        nAtomsPerElement = self.configuration["atom_selection"].get_natoms()
-        for element, number in nAtomsPerElement.items():
-            self._outputData[f"rmsd_{element}"][:] /= number
+        self.configuration["grouping_level"].add_grouped_totals(
+            self._outputData,
+            "rmsd",
+            "LineOutputVariable",
+            axis="time",
+            units="nm",
+            scaling_factor=False,
+            post_func=lambda x: np.sqrt(x / n_atms),
+            post_label="all",
+            main_result=True,
+        )
 
+        nAtomsPerElement = self.configuration["atom_selection"].get_natoms()
         for element, number in nAtomsPerElement.items():
             self._outputData[f"rmsd_{element}"][:] = np.sqrt(
-                self._outputData[f"rmsd_{element}"]
+                self._outputData[f"rmsd_{element}"] / number
```

Comment:
I realise it's in the original, but `number` is a little vague. `natoms`? `n_elem`?